lib/storefront/moon-with-star.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="34px" height="32px" viewBox="0 0 34 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 44.1 (41455) - http://www.bohemiancoding.com/sketch -->
+    <title>100nt</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Main-Core-Mattress" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="1366-Mattress-PDP-EN-Copy" transform="translate(-70.000000, -590.000000)" fill="#00237E">
```

Comment:
so, the idea of icons is that it's purely a symbol, and that we use `fill` css to control the color of the icons. as such, we should take any specific color fills out

Also, let's update the `<desc>` tag with something related to the icon :)